website/src/data/components/time-range/props.js

```diff
@@ -29,20 +29,6 @@ const props = [
         type: 'object[]',
         required: true,
     },
-    {
-        key: 'from',
-        group: 'Base',
-        help: 'start date',
-        type: 'string | Date',
-        required: true,
-    },
-    {
-        key: 'to',
-        group: 'Base',
-        help: 'end date',
-        type: 'string | Date',
-        required: true,
-    },
```

Comment:
Thanks for removing my copy/paste error. 👍

Reply:
No problem! 👍